components/atom/checkbox/src/index.js

```diff
@@ -0,0 +1,50 @@
+import React, {Fragment} from 'react'
+import PropTypes from 'prop-types'
+
+const AtomCheckbox = ({id, value, disabled, checked, onChange, ...props}) => {
```

Comment:
I think that onChange should be required or just has an empty arrow function as a default value